src/topology/instances/real.lean

```diff
@@ -2,28 +2,13 @@
 Copyright (c) 2017 Johannes Hölzl. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Johannes Hölzl, Mario Carneiro
-
-The real numbers ℝ.
-
-They are constructed as the topological completion of ℚ. With the following steps:
```

Comment:
Did you remove this intentionally? Shouldn't it be moved into a module docstring?

Reply:
It was about what another file used to contain.